src/HerbCore.jl

```diff
@@ -6,6 +6,13 @@ include("grammar.jl")
 include("rulenode.jl")
 include("constraint.jl")
 
+const HOLE_NAMES = Dict()
+
+function __init__()
+    HOLE_NAMES[:hole] = Hole
+    HOLE_NAMES[:fshole] = UniformHole
```

Comment:
Maybe a very naive question, but what does `fshole` stand for?

Reply:
It's a relic of when they were called `FixedShape` holes. Now that you mention it, maybe we should update that first 🤔 

It would probably make more sense if every hole used its own name verbatim: `Hole` prints out as `Hole[0, 0, 1]`, not `hole[0, 0, 1]`, for example.